### DISCUSSION TITLE
Преимущества над Module Federation

### DISCUSSION THREAD

**Discussion by vkhv** [TinkoffCreditSystems/shared-library-webpack-plugin#18] [Q&A]
Есть какие-то преимущества над module federation?

**Comment by IKatsuba** [comment]
Спасибо за хороший вопрос!
Главное преимущество плагина - поддержка webpack 4. Предполагаю, что при переезде на webpack 5 сохранится интерфейс взаимодействия, но под капотом будет использовать Module Federation API.
